### ISSUE TITLE
Crash on both Windows system and Linux system

### ISSUE THREAD

**Issue by mengyuefeitian** [lightspark/lightspark#758]
## Crash on both Windows system and Linux system
- website1:http://www.wsbedu.com/wu51/flash.asp
- website2:https://politics.cntv.cn/special/gwyvideo/likeqiang/202011/2020112504/index.shtml
- website3:http://www.4399.com/flash/18012.htm

-https://browser.360.cn/se/ 
This is a browser that always supports NP plug-ins, but currently only 32bit; although it comes with flash29 itself, but does not support Linux, we still look forward to a better solution and support all platforms
---

We are looking for a solution to the content breakdown of these websites. We are willing to donate to you and look forward to contact。 mengyuefeitian#gmail.com

**Comment by eiglow** [comment]
Are you able to get direct links to the .swf files on those websites? Because I'm having difficulty finding them.

Either a direct link, or a file upload (as a zip) will work.

**Comment by mengyuefeitian** [comment]
> Are you able to get direct links to the .swf files on those websites? Because I'm having difficulty finding them.
> 
> Either a direct link, or a file upload (as a zip) will work.

[swf.zip](https://github.com/lightspark/lightspark/files/6690743/swf.zip)
I can get two swf files, website3 is an online flash game, I cannot get it, I can only provide the front-end source code.
We hope to get your help.

**Comment by eiglow** [comment]
For the first swf, I have opened an issue: #763 
For the second swf, it appears to need a video id param. I use lightspark standalone with a terminal, so I need to set the param via `-p` to test this properly. However, I'm not sure how exactly `-p` works, so I opened a Q&A discussion: #765
For the third swf, I think I managed to get it. I downloaded the files it tried to download, and placed them where it expects to find them. I have opened an issue: #766
